lib/registration.pm

```diff
@@ -201,6 +201,7 @@ sub fill_in_registration_data {
             assert_screen("module-selection");
             send_key $cmd{next};
         }
+        else {send_key $cmd{next};}
```

Comment:
that does not make any sense. If the same key is pressed in both "if" as well as "else", it should be outside the if-else.
OTOH why? ;-)